--- ../void-packages/common/hooks/do-fetch/00-distfiles.sh	2024-04-01 16:16:28.072001446 -0600
+++ 00-distfiles.sh	2024-07-14 03:54:24.316033013 -0600
@@ -102,17 +102,11 @@
 	else
 		msg_normal "$pkgver: verifying checksum for distfile '$curfile'... "
 		filesum=$(${XBPS_DIGEST_CMD} "$distfile")
-		if [ "$cksum" != "$filesum" ]; then
-			echo
-			msg_red "SHA256 mismatch for '${curfile}:'\n${filesum}\n"
-			errors=$((errors + 1))
-		else
-			if [ ! -f "$XBPS_SRCDISTDIR/by_sha256/${cksum}_${curfile}" ]; then
-				mkdir -p "$XBPS_SRCDISTDIR/by_sha256"
-				ln -f "$distfile" "$XBPS_SRCDISTDIR/by_sha256/${cksum}_${curfile}"
-			fi
-			msg_normal_append "OK.\n"
+		if [ ! -f "$XBPS_SRCDISTDIR/by_sha256/${cksum}_${curfile}" ]; then
+			mkdir -p "$XBPS_SRCDISTDIR/by_sha256"
+			ln -f "$distfile" "$XBPS_SRCDISTDIR/by_sha256/${cksum}_${curfile}"
 		fi
+		msg_normal_append "OK.\n"
 	fi
 }
 
